### DISCUSSION TITLE
useResult Function?

### DISCUSSION THREAD

**Discussion by bobmcallan** [urql-graphql/urql#1703] [Q&A]
Does urql provide a function similar to Apollo's useResult. If not is this achievable through plugins or such.

Thanks,
Bob

**Comment by JoviDeCroock** [comment]
For reference this is a Vue-API question and `useResult` is defined [here](https://v4.apollo.vuejs.org/api/use-result.html).

To answer your question as I'm not highly familiar with Vue this looks like a utility to me that you could probably make yourself as it just needs the `QueryResult`, a `defaultValue` and then a way to mask data.

**Reply by kitten** [reply]
If I'm not mistaken, you'd want to instead use `computed`, which is built into Vue 3's reactivity system: https://v3.vuejs.org/guide/reactivity-computed-watchers.html#computed-values

**Reply by bobmcallan** [reply]
Thanks for the reply's. I've currently implemented with the compute function, the useResult was a little cleaner is all.

Bob
